packages/react/src/components/stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,307 @@
+import React, { useReducer } from 'react';
+
+import { Stepper } from './Stepper';
+import styles from './Stepper.module.scss';
+import { Step } from './Step';
+import { Button } from '../button';
+
+export default {
+  component: Stepper,
+  title: 'Components/Stepper',
+  decorators: [(storyFn) => <div style={{ maxWidth: '700px' }}>{storyFn()}</div>],
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+const commonReducer = (stepsTotal) => (state, action) => {
+  switch (action.type) {
+    case 'completeStep': {
+      const activeStep = action.payload === stepsTotal ? stepsTotal : action.payload + 1;
+      return {
+        activeStep,
+        states: state.states.map((stateName, index) => {
+          if (index === action.payload - 1 && index !== stepsTotal - 1) {
+            // current one but not last one
+            return 'filled';
+          }
+          if (index === action.payload) {
+            // next one
+            return 'available';
+          }
+          return stateName;
+        }),
+      };
+    }
+    case 'setActive': {
+      return {
+        activeStep: action.payload,
+        states: state.states.map((stateName, index) => {
+          if (index === action.payload - 1) {
+            return 'available';
+          }
+          return stateName;
+        }),
+      };
+    }
+    default:
+      throw new Error();
+  }
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Default = (args) => {
```

Comment:
On mobile this does not look too good - the left padding seems way too big. Is this bigger than it actually is due to the Storybook frame?

![mobile-padding](https://user-images.githubusercontent.com/4214671/151327684-224b9ae4-1c72-42b3-888a-dd6981019b8c.jpg)

Reply:
True, it is bigger because of the storybook frame. Still, I also don't like the fact that we can't align it fully to the left edge. If people want to align their content based on the stepper, the alignment will also be very hard to get right. Since if you use different languages, the alignment would be different for each of them. 

I wonder if we should use the KELA model, where the line would extend to left edge? Atleast we would get better alignment then. Other option would be to somehow try to align the first step to left, but that would screw up other things, like the mutual distances between steps.

Reply:
- [ ] Apply negative margin to fix horizontal alignment